printer.py

```diff
@@ -59,6 +60,11 @@ def move(direction):
 def status(size):
     print("Your katamari is {0}cm".format(str(size)))
 
+def bump(item):
+    status=["bumps and","rubs and","wiggles then","stalls then"]
+    verb = status[random.randrange(len(status))]
+    print("Your katamari {0} rolls back from the {1} before slowing".format(verb, item))
```

Comment:
I'd add something like "looks like you're not big enough to pick up this item yet" to the end